src/webcomponents/workflow/workflow-import.js

```diff
@@ -92,35 +92,15 @@ export default class WorkflowImport extends LitElement {
     }
 
     async fetchRepositories(org) {
-        const url = `https://api.github.com/orgs/${org}/repos`;
-        const headers = {
-            // 'Authorization': `token ${token}`,
-            "Accept": "application/vnd.github.v3+json"
-        };
-
         this.repositories = [];
-        let page = 1;
-        try {
-            while (true) {
-                const response = await fetch(`${url}?page=${page}&per_page=100`, {headers});
-                if (!response?.ok) {
-                    throw new Error(`Error fetching repositories: ${response?.statusText}`);
-                }
 
-                let data = await response.json();
-                data = data
-                    .filter(repo => repo.name !== "tools")
-                    .filter(repo => !repo.archived);
-                    // Note (Nacho 21/03/25): filter commented to show PacVar repository
-                    // .filter(repo => repo.topics.includes("nf-core") && repo.topics.includes("workflow"));
-                if (data.length === 0) {
-                    break;
-                } // No more repositories
-
-                this.repositories = this.repositories.concat(data);
-                page++;
+        try {
```

Comment:
The `fetchRepositories` method doesn’t need to be called during opencgaSession update

Reply:
Update - Fixed